pysat/tests/test_utils_coords.py

```diff
@@ -43,6 +43,7 @@ def setup(self):
         """Runs after every method to clean up previous testing."""
         self.py_inst = None
         self.inst_time = dt.datetime(2009, 1, 1)
+        self.inst_2_time = dt.datetime(2009, 1, 3)
```

Comment:
```suggestion
        self.inst_time_2 = dt.datetime(2009, 1, 3)
```

Because it's the second instrument time, not the second instrument's time.